src/Agent.Listener/LocalRunner.cs

```diff
@@ -133,22 +141,35 @@ public sealed override void Initialize(IHostContext hostContext)
             int requestId = 1;
             foreach (Phase phase in process.Phases ?? new List<IPhase>(0))
             {
-                foreach (Job job in phase.Jobs ?? new List<IJob>(0))
+                IDictionary<string, IDictionary<string, string>> matrix;
```

Comment:
I need to cleanup and merge my other PR (command line cleanup) and then add support for specifying the phase/job from the command line. I'm planning to make that change in a future PR.